docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  eventsapi:
+    build:
+      context: .
+    volumes:
+      - .:/code
+    ports:
+      - "9000:80"
```

Comment:
@mjoss17 , @P-Gill97 - Any thoughts on choice of this port? I skipped to `9xxx` since we already use so many in the `8xxx` space for other services typically running in our dev environments, and I didn't want to think too hard about a value that wouldn't collide :joy: . It's pretty arbitrary so I'm open to alternate suggestions.